.gitlab-ci.yml

```diff
@@ -66,14 +93,14 @@ notify-slack:
 
 
 notify-failed-pipeline:
-  stage: notify
-  image: $NOTIFIER_IMAGE
-  when: on_failure
-  script:
-    - |
-      MESSAGE="The pipeline uncountered an unexpected error."
-      postmessage "$NOTIFICATIONS_SLACK_CHANNEL" "$MESSAGE" alert
-  tags: [ "runner:main", "size:large" ]
+ stage: notify
+ image: $NOTIFIER_IMAGE
+ when: on_failure
+ script:
+   - |
+     MESSAGE="The pipeline uncountered an unexpected error."
+     postmessage "$NOTIFICATIONS_SLACK_CHANNEL" "$MESSAGE" alert
+ tags: [ "runner:main", "size:large" ]
```

Comment:
Is there a reason to indent this block with one space instead of 2?

Reply:
This was marked as solved, but is there a reason?